classes/03class/exercises/c03-make01/gpavelar/Makefile

```diff
@@ -0,0 +1,25 @@
+REGISTRY ?= $$(docker info | sed '/Username:/!d;s/.* //' |  sed "s/ //g")
+IMAGE ?= make01
+TAG ?= $$(git rev-parse --short HEAD)
+
+
+
+.PHONY: build
+build:
+	@echo ${REGISTRY}
+
+	@echo "Building Docker Image ${REGISTRY}/${IMAGE}:${TAG}"
+	@docker build -t ${IMAGE}:${TAG} \
+		--build-arg REGISTRY=${REGISTRY} \
+		--build-arg TAG=${TAG} .
+
+.PHONY: run
+run:
+	@echo "Running the container on port 8081..."
+	@docker run -d -p 8081:80 ${IMAGE}:${TAG}
```

Comment:
The port should be passed as parameter here:

> The port to be mapped for the container should be dynamically provided during the make run command.

Reply:
oh no, I will change it.